### PR TITLE
Fix publish pipeline

### PR DESCRIPTION
**Pull request by carryall**
## What happened 👀

[The release pipeline](https://github.com/nimblehq/stylelint-config-nimble/actions/runs/5452909247/jobs/9921049358) was broken. This fix upgrades the node version but not the Lerna version yet (the latest version is 7.1.1).

## Insight 📝

N/A

## Proof Of Work 📹

Test by triggering the event and [it's success](https://github.com/nimblehq/stylelint-config-nimble/actions/runs/5455775561/jobs/9927685846?pr=16)
Screenshot 2566-07-04 at 21 21 30

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   release:
     types:
       - published
+  push:
+    branches:
+      - "bug/fix-publish-pipeline"
```
Comment (hoangmirs) [inline .github/workflows/publish.yml]: Should we remove this? 😄

Comment (carryall) [reply .github/workflows/publish.yml]: Actually, I already pushed a commit to remove it 🙈 but not sure why it's not here
Screenshot 2566-07-05 at 09 35 28

Comment (carryall) [reply .github/workflows/publish.yml]: Fixed by push an extra commit 😂

**Review by malparty — APPROVED** [review]

**Review by hoangmirs — APPROVED** [review]
Could you clone this for eslint-config as well? 🙏

**Comment by manh-t** [comment]
@carryall since we update node version for the workflows. Can we update it for the workflow `lint` too? 🙏 
image

**Comment by carryall** [comment]
> @carryall since we update node version for the workflows. Can we update it for the workflow `lint` too? 🙏 image

Thanks @manh-t , done in ca18e44

**Review by manh-t — APPROVED** [review]

**Pull request merged**